doc/manual/configuration.rst

```diff
@@ -746,6 +746,17 @@ of the respective attributes of the recipe. For example the scripts of the
 inherited class of all steps are inserted in front of the scripts of the
 current recipe. 
 
+metaEnvironment
+~~~~~~~~~~~~~~~
+
+Type: Dictionary (String -> String)
+
+metaEnvironment variables behave like :ref:`configuration-recipes-privateenv` variables.
+They overrule other environment variables and can be used in all steps. In addition all
+metaEnvironment variables are added to the audit no matter they are used in a step or not.
+This predestines metaEnvironment variables to add the license type or version of a package
```

Comment:
Missing period at end of sentence. Maybe we should mention that variable substitution is not available...